pkg/analytics/executions.go

```diff
@@ -0,0 +1,95 @@
+package analytics
+
+import (
+	"github.com/bacalhau-project/bacalhau/pkg/models"
+)
+
+const ExecutionEventType = "bacalhau.execution_v1"
+
+type ExecutionEvent struct {
+	ExecutionID string `json:"execution_id,omitempty"`
+	JobID       string `json:"job_id,omitempty"`
+	EvalID      string `json:"eval_id,omitempty"`
+
+	// name of execution
+	Name string `json:"name,omitempty"`
+	// hash of the node name running execution
+	NodeNameHash string `json:"node_name_hash,omitempty"`
+	// hash of namespace execution is running in
+	NamespaceHash string `json:"namespace_hash,omitempty"`
+
+	// AllocatedResources
+	// NB: total execution resources for all tasks in execution
+	TotalCPUUnits    float64 `json:"total_cpu_units,omitempty"`
+	TotalMemoryBytes uint64  `json:"total_memory_bytes,omitempty"`
+	TotalDiskBytes   uint64  `json:"total_disk_bytes,omitempty"`
+	TotalGPUCount    uint64  `json:"total_gpu_count,omitempty"`
+
+	DesiredState        string `json:"desired_state,omitempty"`
+	DesiredStateMessage string `json:"desired_state_message,omitempty"`
+	ComputeState        string `json:"compute_state,omitempty"`
+	ComputeStateMessage string `json:"compute_state_message,omitempty"`
+
+	PublishedResultType string `json:"published_result_type,omitempty"`
```

Comment:
isn't this the same as `publisher_type`? I mean you can publish it along with engine types and input source types that you publish with job event, but just be consistent about field naming